information-schema/information-schema-data-lock-waits.md

```diff
@@ -22,24 +18,50 @@ DESC data_lock_waits;
 +------------------------+---------------------+------+------+---------+-------+
 | Field                  | Type                | Null | Key  | Default | Extra |
 +------------------------+---------------------+------+------+---------+-------+
-| KEY                    | varchar(64)         | NO   |      | NULL    |       |
+| KEY                    | text                | NO   |      | NULL    |       |
+| KEY_INFO               | text                | YES  |      | NULL    |       |
 | TRX_ID                 | bigint(21) unsigned | NO   |      | NULL    |       |
 | CURRENT_HOLDING_TRX_ID | bigint(21) unsigned | NO   |      | NULL    |       |
 | SQL_DIGEST             | varchar(64)         | YES  |      | NULL    |       |
+| SQL_DIGEST_TEXT        | text                | YES  |      | NULL    |       |
 +------------------------+---------------------+------+------+---------+-------+
 ```
 
 `DATA_LOCK_WAITS` 表中各列的字段含义如下：
 
-* `KEY`：正在发生等锁的 KEY，以十六进制编码的形式显示。
+* `KEY`：正在发生等锁的 key，以十六进制编码的形式显示。
+* `KEY_INFO`：对 `KEY` 进行解读得出的一些详细信息，见 [KEY_INFO](#key_info)。
 * `TRX_ID`：正在等锁的事务 ID，即 `start_ts`。
 * `CURRENT_HOLDING_TRX_ID`：当前持有锁的事务 ID，即 `start_ts`。
 * `SQL_DIGEST`：当前正在等锁的事务中被阻塞的 SQL 语句的 Digest。
+* `SQL_DIGEST_TEXT`：当前正在等锁的事务中被阻塞的 SQL 语句的归一化形式，即去除了参数和格式的 SQL 语句。与 `SQL_DIGEST` 对应。
 
 > **警告：**
 >
-> * 该表中的信息是在查询时，从所有 TiKV 节点实时获取的。目前，即使加上了 `WHERE` 查询条件，也无法避免对所有 TiKV 节点都进行信息收集。如果集群规模很大、负载很高，查询该表有造成性能抖动的潜在风险，因此请根据实际情况使用。
-> * 来自不同 TiKV 节点的信息不一定是同一时间点的快照。
+> * 仅拥有 [PROCESS](https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html#priv_process) 权限的用户可以查询该表。
+> * `DATA_LOCK_WAITS` 表中的信息是在查询时，从所有 TiKV 节点实时获取的。目前，即使加上了 `WHERE` 查询条件，也无法避免对所有 TiKV 节点都进行信息收集。如果集群规模很大、负载很高，查询该表有造成性能抖动的潜在风险，因此请根据实际情况使用。
+> * 来自不同 TiKV 节点的信息不能保证是同一时间点的快照。
+> * `SQL_DIGEST` 列中的信息（SQL Digest）为 SQL 语句进行归一化后计算得到的 hash。`SQL_DIGEST_TEXT` 列中的信息为内部从 Statements Summary 系列表中查询得到，因而存在内部查询不到对应语句的可能性。关于 SQL Digest 和 Statements Summary 相关表的详细说明，请参阅[Statement Summary Tables](/statement-summary-tables.md)。
+
+## `KEY_INFO`
+
+`KEY_INFO` 列中展示了对 `KEY` 列中所给出的 key 的详细信息，以 JSON 格式给出。其包含的信息如下：
+
+* `"db_id"`：该 key 所属的数据库（schema）的 ID。
+* `"db_name"`：该 key 所属的数据库（schema）的名称。
+* `"table_id"`：该 key 所属的表的 ID。
+* `"table_name"`：该 key 所属的表的名称。
+* `"handle_type"`：该 row key 的 handle 类型，其可能的值有：
+    * `"int"`：handle 为 int 类型，即 handle 为 row id；
+    * `"common"`：非 int64 类型的 handle，在启用 clustered index 时非 int 类型的主键会显示为此类型；
+    * `"unknown"`：当前暂不支持的 handle 类型。
+* `"partition_handle"`：是否为 partition handle。
```

Comment:
Here's some misunderstanding of mine about the `partition_handle` field. I'll fix it next week.